### ISSUE TITLE
Fix issues in chrome debugger

### ISSUE THREAD

**Issue by Jexxer** [Jexxer/DepositCalculator#16]
The browser in chrome gives some insights into missing tag fields. Correct these so we can set a clean slate and ensure that we can maintain clean accessible code.

![Image](https://github.com/Jexxer/DepositCalculator/assets/81524053/7ae7a1dc-7642-4f0f-bea1-53cd3091c93e)
